code-reviews/recipes/Bash.md

```diff
@@ -41,4 +134,9 @@ fi
 
 In addition to the [Code Review Checklist](../README.md) you should also look for these bash specific code review items
 
-> TODO: Add checklist
+* [ ] Does this code use [Built-in Shell](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) Options like set -o, set -e, set -u for execution control of shell scripts.
```

Comment:
Nit: This should end in a question mark.

Reply:
Done

Reply:
@ChrisRisner , made changes, can you please review and approve the PR.